src/azure-cli/azure/cli/command_modules/eventhubs/_params.py

```diff
@@ -56,6 +56,10 @@ def load_arguments_eh(self, _):
         c.argument('require_infrastructure_encryption', options_list=['--infra-encryption'], is_preview=True,
                    arg_type=get_three_state_flag(),
                    help='A boolean value that indicates whether Infrastructure Encryption (Double Encryption) is enabled/disabled')
+        c.argument('public_network_access', options_list=['--public-network-access', '--public-network'],
+                   arg_type=get_enum_type(['Enabled', 'Disabled']),
```

Comment:
Can you use the enum defined from SDK instead of manually hardcode `['Enabled', 'Disabled']`?

Reply:
@evelyn-ys, we would like to remove our dependency from SDK here. We can declare these enums as constants if required.